packages/modular-scripts/src/start.ts

```diff
@@ -57,14 +55,16 @@ async function start(packageName: string): Promise<void> {
     );
   }
 
+  let paths;
   const isEsmView = isModularType(targetPath, 'esm-view');
   const isView = isModularType(targetPath, 'view');
   if (isView) {
     targetPath = stageView(target);
+    paths = determineTargetPaths(target, targetPath);
```

Comment:
`paths = determineTargetPaths(target, targetPath);` in the if branch is repeated in the else branch. Maybe it just needs to be outside the if?

Reply:
Target is updated in one of the two if branches, thus changing the output of determine target branches 
It was actually a bug that I took a long time to figure out

Reply:
I see. We'll need to disentangle this code, but not for now. Can you put a `//TODO: this needs to be rewritten in a clearer way` or something equivalent?

Reply:
Now that I've revisited it I cleaned it up